api/filters_test.go

```diff
@@ -1180,13 +1203,15 @@ func TestRequestForwardingMiddleware(t *testing.T) {
 					Dataset: &models.Dataset{
 						Version: 1,
 					},
+					Type: "not-flexible",
```

Comment:
Test name indicates a flexible filter type but the mock returns Type: "not-flexible"

Reply:
If you expand the bar closest above to this comment, you'll see that it relates to the appropriate  `convey` statement